src/main/java/de/unistuttgart/towercrushbackend/data/Configuration.java

```diff
@@ -38,7 +38,7 @@ public void removeQuestion(final Question question) {
     }
 
     @Override
-    public Configuration clone() {
+    public Configuration clone() { //NOSONAR
```

Comment:
Not needed here?

Reply:
```suggestion
    public Configuration clone() {
```

Reply:
Was neede because java:S2975, but deactivated it..